examples/llm/inference/hf_generate.py

```diff
@@ -65,7 +67,19 @@ def parse_args() -> Namespace:
                         nargs='?',
                         const=True,
                         default=True)
+    parser.add_argument('--trust_remote_code',
+                        type=str2bool,
+                        nargs='?',
+                        const=True,
+                        default=True)
+    parser.add_argument('--use_auth_token',
```

Comment:
I think `use_auth_token` is actually supposed to be the token value itself? At least that is what I had been doing...

Reply:
It can be optionally a string, or if set to true, it will look in the environment so either `HUGGING_FACE_HUB_TOKEN` must be set as a envvar, or you have to have logged in via `huggingface-cli login`

Should I also allow a string to be passed in? I can do it but ill have to make a new dtype str_or_bool :P

Reply:
@dakinggg I made it possible to pass str_or_bool, and also added a better error message that explains how to set the env var